packages/flutter/test/material/dialog_test.dart

```diff
@@ -85,6 +85,24 @@ void main() {
     expect(materialWidget.shape, _defaultDialogShape);
   });
 
+  testWidgets('Custom dialog elevation', (WidgetTester tester) async {
+    const double customElevation = 12.0;
+    const AlertDialog dialog = AlertDialog(
+      actions: <Widget>[ ],
+      elevation: customElevation,
+    );
+    await tester.pumpWidget(_appWithAlertDialog(tester, dialog));
+
+    await tester.tap(find.text('X'));
+    await tester.pump(); // start animation
+    await tester.pump(const Duration(seconds: 1));
+
+    final StatefulElement widget = tester.element(
```

Comment:
This idiom appears several times in this file. Might be good to define a function that returns a Dialog's Material